components/skeleton/style/index.less

```diff
@@ -168,6 +175,24 @@
   }
 }
 
+// List
+.skeleton-element-list() {
+  display: inline-block;
+  vertical-align: top;
+  background: @skeleton-color;
+  border-radius: @border-radius-base;
+
+  .skeleton-element-list-size(@btn-height-base);
```

Comment:
Please don't use btn's style. You can use List component const or you define it yourself.

Reply:
okay will do it

Reply:
@Rustin-Liu I have use the same CSS used in List component?

Reply:
Yes